test-data/src/test/resources/org/openmrs/module/fhir2/providers/TestTask_update.xml

```diff
@@ -0,0 +1,6 @@
+<Task xmlns="http://hl7.org/fhir">
```

Comment:
Is this file actually used anywhere?

Reply:
I think yes, rather am not sure of how its gonna be written to refer to json file